asf_tools/flood_map.py

```diff
@@ -1,3 +1,12 @@
+"""Generate flood depth map from surface water extent map.
+
+Create a flood depth map from a surface water extent map and
+a HAND image. The HAND image must be pixel-aligned (same extent and size) to
+the water extent map, and the surface water extent map should only have one class
+of water. Flood depth maps are estimated using either a numerical, normalized
```

Comment:
> the surface water extent map should only have one class of water.

Might be better to say "the surface water extent map should be a byte GeoTIFF indicating water (true), not water (false)." to be specific -- I'm not sure if "water classes" would immediately be recognized outside of the HydroSAR team.